example/config/otherchecks/timedchecks.js

```diff
@@ -4,43 +4,42 @@
 require('isomorphic-fetch');
 
 let lastCheckOutput = 'Waiting for check',
-    lastCheckTime,
-    panicGuide = '`Don\'t panic`';
+  lastCheckTime,
+  panicGuide = "`Don't panic`";
```

Comment:
Are the ticks (`) intentional? It's cool if they are.

Reply:
That's a question for @codeclout